### PR TITLE
Adds ActionList on the Home feed driven by a Rock content channel. 

### LINKED ISSUE #800: Hook up existing "Today Actions" feature to Rock / new Features framework
HMW model the today list? - Apollos for Apollos

Content should be configurable via Rock.

### PR DESCRIPTION
**Pull request by vinnyjth**
## DESCRIPTION

Blocked by #828 

### What does this PR do, or why is it needed?

Takes all the API work I've done, and applies it to the front end. Now the ActionLists on the home feed are driven by a persona feed and a rock content channel. This is all configurable via the API.

Closes #800 
### What design trade-offs/decisions were made?

In a future PR, I will take the `ContentTableCard` component and move it to the UI-kit. 

### How do I test this PR?

1. Open the app.
2. Check out the sweet new content on the home feed.
- Click on them and make sure they open up content. 

### What automated tests did you write?

n/a

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes #800 
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/tabs/home/Home.js [hunk 1]
```diff
@@ -117,37 +117,40 @@ class Home extends PureComponent {
...
-                            'personaFeed.edges',
-                            []
-                          ).map((edge) => edge.node)}
-                        />
-                      )}
+                        data: features,
+                        loading: featuresLoading,
+                        ...other
+                      }) =>
+                        console.log(features, other) ||
```
Comment (redreceipt) [inline packages/apolloschurchapp/src/tabs/home/Home.js]: Take this little guy out. If you give me a thumbs up, I'll do it.

File: packages/apolloschurchapp/src/ui/ContentTableCard/index.js [hunk 2]
```diff
@@ -41,10 +41,10 @@ class ContentTableCard extends PureComponent {
               isLoading={isLoading}
               key={item.id}
               id={item.id}
-              onPress={onPress}
-              label={item.parentChannel ? item.parentChannel.name : ''}
+              onPress={() => onPress(item)}
+              label={item.subtitle || ''}
               title={item.title || ''}
-              imageSource={item.coverImage ? item.coverImage.sources : ''}
+              imageSource={item.image ? item.image.sources : ''}
```
Comment (redreceipt) [inline packages/apolloschurchapp/src/ui/ContentTableCard/index.js]: I like this SOO much better than using lodash!!

**Comment by redreceipt** [comment]
Not sure what I'm looking for here or what it's supposed to do. Plus, clicking the new content doesn't seem to do anything for me.


Kapture 2019-06-30 at 13 53 36

**Comment by redreceipt** [comment]
> Not sure what I'm looking for here or what it's supposed to do. Plus, clicking the new content doesn't seem to do anything for me.
> 
> Kapture 2019-06-30 at 13 53 36

Ah, I bet I needed that last pull from `master`, hang on I'll try again...

**Comment by redreceipt** [comment]
Nah, same thing. Still doesn't seem to be doing anything.

**Comment by vinnyjth** [comment]
@redreceipt Did you `yarn` before starting this PR?

**Comment by vinnyjth** [comment]
After @conrad-vanl gives this a once over, we'll merge.

File: packages/apolloschurchapp/src/tabs/home/Home.js (hunk 1 above)
Comment (conrad-vanl) [reply packages/apolloschurchapp/src/tabs/home/Home.js]: Yeah, lets take this out
```suggestion

```

File: packages/apolloschurchapp/src/tabs/home/Home.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ class Home extends PureComponent {
 
   handleOnPress = (item) =>
     this.props.navigation.navigate('ContentSingle', {
-      itemId: item.id,
+      itemId: item.relatedNode.id,
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/tabs/home/Home.js]: Does this change not break the other elements on the screen that call this function, or does they all call this with `item.relatedNode` being set now?

Comment (vinnyjth) [reply packages/apolloschurchapp/src/tabs/home/Home.js]: DOH! Good catch @conrad-vanl

**Pull request merged**